### ISSUE TITLE
Native ipv6 support?

### ISSUE THREAD

**Issue by hathlife** [FWGS/xash3d#375]
Ipv6 network has been widely availabled in US , EU, Japan,and soon China, etc.with ipv6 support ones can set up his/her own game  server easily, without worrying about "sh*t I don't even have a public IP address / I'm using mobile network but still want to set up a temporary server on my phone to play".
However original goldsrc& source games doesn't support ipv6.so it's expensive to set up a server in ipv4-address-short country like China.
I think it's a important feature for future.

**Comment by mittorn** [comment]
We have an idea how to implement it and leave compatible with network api, but not implemented yet

**Comment by a1batross** [comment]
Yes, netadr_t structure from netadr.h(which is public in HLSDK) can handle IPv6 address and port. 

The only thing is it's needs time to be implemented. And master query protocol should be updated for IPv6.
